sys/mips/pmap.c

```diff
@@ -509,6 +510,7 @@ static void pmap_setup(pmap_t *pmap) {
 void init_pmap(void) {
   pmap_setup(&kernel_pmap);
   kernel_pmap.pde = _kernel_pmap_pde;
+  pmap_maxkvaddr = rounddown((vaddr_t)__kernel_end, PAGESIZE);
```

Comment:
Why are you rounding down and not up?

Reply:
I think that it doesn't matter as long as last used page will be mapped.
But `roundup` is better here - we don't want to have anything after end - I changed that.